cloudpathlib/cloudpath.py

```diff
@@ -576,6 +576,8 @@ def read_text(self):
     def download_to(self, destination: Union[str, os.PathLike]):
         destination = Path(destination)
         if self.is_file():
+            if destination.is_dir():
+                destination = destination.joinpath(self.name)
```

Comment:
```suggestion
                destination = destination / self.name
```

Rather than `joinpath`, we prefer the `/` operator style of using pathlib.